nebula-logger/main/logger-engine/classes/FlowLogEntry.cls

```diff
@@ -59,6 +65,15 @@ global inherited sharing class FlowLogEntry {
     global static List<String> addFlowEntries(List<FlowLogEntry> flowLogEntries) {
         Boolean saveLog = false;
         for (FlowLogEntry flowLogEntry : flowLogEntries) {
+            // Set the logging level if it's blank
```

Comment:
It almost feels like this method should be put into the `LogEntryEventBuilder` and exposed as a public method so that  `FlowRecordLogEntry` and this can share the logic without it being maintained in both places.

Reply:
I definitely agree, I don't like all of the code duplication. But I'll handle that change later - right now, everything is internal code, so I want to look into it a bit more before creating any new `public` methods.